projects/fine_tuning/toolbox/fine_tuning_run_fine_tuning_job/tasks/main.yml

```diff
@@ -362,3 +362,37 @@ elif fine_tuning_run_fine_tuning_job_workload == 'ilab' -%}
          > {{ artifact_extra_logs_dir }}/artifacts/pod_{{ item | replace('pod/', '') }}.log
     failed_when: false
     loop: "{{ job_pod_name_cmd.stdout_lines }}"
+
+  - name: Retrieve Pod files to artifacts directory
+    when: fine_tuning_run_fine_tuning_job_retrieve_files | bool
+    block:
+      - name: Create the Pod artifacts directory
+        file:
+          path: "{{ artifact_extra_logs_dir }}/pod-artifacts/"
+          state: directory
+          mode: '0755'
+
+      - name: Launch fine-tuning-file-retrieval Pod from template
+        command:
+          oc create -f <(echo "{{ fine_tuning_job_retrieval_pod_template | to_json }}")
+
+      - name: Wait for fine-tuning-file-retrieval Pod to be Running
+        command:
+          set -o pipefail;
+
+          oc get pod --no-headers
+             fine-tuning-file-retrieval
+             -n {{ fine_tuning_run_fine_tuning_job_namespace }}
```

Comment:
here you can use `oc get -f "{{ artifact_extra_logs_dir }}/src/revrieval_pod.yaml"` and remove the pod name & namespace